### ISSUE TITLE
Add licensing lines to all the code files

### ISSUE THREAD

**Issue by IllyaMoskvin** [roundware/roundware-server#284]
Many files are missing the requisite licensing declaration @hburgund mentioned in #283. I think it makes sense to treat this as a separate issue, go through all the files, and create a separate pull request.

```
# Roundware Server is released under the GNU Affero General Public License v3.
# See COPYRIGHT.txt, AUTHORS.txt, and LICENSE.txt in the project root directory.
```

**Comment by hburgund** [comment]
Good idea to do a full audit of all files, but please still add the lines to the new files you create in #283 so they are properly licensed right from the start.

**Comment by 13rac1** [comment]
A few py files missing the AGPL notice:
- https://github.com/roundware/roundware-server/blob/db3a02d41415ab8f39c41a0c7d0afd031a2c4c8c/roundware/rw/mommy_recipes.py
- https://github.com/roundware/roundware-server/blob/db3a02d41415ab8f39c41a0c7d0afd031a2c4c8c/roundware/notifications/admin.py
- https://github.com/roundware/roundware-server/blob/db3a02d41415ab8f39c41a0c7d0afd031a2c4c8c/roundware/notifications/admin.py
- https://github.com/roundware/roundware-server/blob/db3a02d41415ab8f39c41a0c7d0afd031a2c4c8c/roundware/lib/api.py
- https://github.com/roundware/roundware-server/blob/db3a02d41415ab8f39c41a0c7d0afd031a2c4c8c/roundware/api2/filters.py
